qsirecon/interfaces/anatomical.py

```diff
@@ -137,15 +134,6 @@ def _run_interface(self, runtime):
             "t1_2_mni_forward_transform",
             "%s/sub-%s*_from-T*w_to-MNI152NLin2009cAsym_mode-image_xfm.h5" % (anat_root, sub),
         )
-        if not self.inputs.infant_mode:
-            self._results["template_image"] = pkgrf(
-                "qsirecon",
-                "data/mni_1mm_t1w_lps_brain.nii.gz",
-            )
-        else:
-            self._results["template_image"] = pkgrf(
-                "qsirecon", "data/mni_1mm_t1w_lps_brain_infant.nii.gz"
-            )
```

Comment:
This step _was_ using skullstripped versions of the templates, but TemplateFlow doesn't have a skullstripped version of the MNIInfant templates (although it has a brain mask I could apply to the non-skullstripped template). @mattcieslak do you think that will be a problem?

Reply:
We definitely want to use skull-stripped images for registration. Maybe we could apply the mask and then reorient?